Observation/Observation-NecrosisKayBurbridge-Example.json

```diff
@@ -0,0 +1,34 @@
+{
+  "resourceType": "Observation",
+  "id": "Observation-NecrosisKayBurbridge-Example",
+  "status": "final",
+  "code": {
+    "coding": [
+      {
+        "system": "http://snomed.info/sct",
+        "code": "6574001",
+        "display": "Necrosis (morphologic abnormality)"
```

Comment:
I would switch this out for "Percentage of neoplasm involved by necrosis (observable entity)
SCTID: 405921002" as we are recording an observable entity rather than the presence of an abnormality (though the original would probably still be interpreted correctly).